src/CFamily.UnitTests/CMake/CMakeSettingsProviderTests.cs

```diff
@@ -21,32 +21,28 @@
 using System;
 using System.IO;
 using System.IO.Abstractions;
-using CMakeSettingsProviderTestsExtensions;
+using System.IO.Abstractions.TestingHelpers;
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
 using Newtonsoft.Json;
 using SonarLint.VisualStudio.CFamily.CMake;
 using SonarLint.VisualStudio.Integration;
 using SonarLint.VisualStudio.Integration.UnitTests;
-using SonarLint.VisualStudio.Integration.UnitTests.Extensions;
 
 namespace SonarLint.VisualStudio.CFamily.UnitTests.CMake
 {
     [TestClass]
     public class CMakeSettingsProviderTests
     {
-        private const string RootDirectory = "dummy root";
+        private const string RootDirectory = "c:\\dummy root";
 
         [TestMethod]
         public void Find_FileDoesNotExist_Null()
         {
-            var cmakeSettingsLocation = GetCmakeSettingsLocation(RootDirectory);
+            var fileSystem = SetupFileSystem();
```

Comment:
This seems to be the only test that relies on the constructed file system having an empty `RootDirectory` folder.
It would be better if the test explicitly created the empty folder it wanted to test.